frontend/javascripts/components/slider.tsx

```diff
@@ -40,49 +44,66 @@ export function Slider(props: SliderProps) {
     step,
     disabled,
   } = props;
+  const isFocused = useRef(false);
+  const sliderRef = useRef<HTMLDivElement>(null);
+
+  const handleWheelEvent = useCallback(
+    (event: { preventDefault: () => void; deltaY: number }) => {
+      // differentiate between single value and range slider
+      if (onWheelDisabled || value == null || min == null || max == null || !isFocused.current)
+        return _.noop;
+      if (range === false || range == null) {
+        event.preventDefault();
+        const newValue = value - getWheelStepFromEvent(ensuredStep, event.deltaY, wheelStep);
+        const clampedNewValue = clamp(min, newValue, max);
+        if (onChange != null) onChange(clampedNewValue);
+      } else if (range === true || typeof range === "object") {
+        event.preventDefault();
+        const diff = getWheelStepFromEvent(ensuredStep, event.deltaY, wheelStep);
+        const newLowerValue = Math.round(value[0] + diff);
+        const newUpperValue = Math.round(value[1] - diff);
+        const clampedNewLowerValue = clamp(min, newLowerValue, Math.min(newUpperValue, max));
+        const clampedNewUpperValue = clamp(newLowerValue, newUpperValue, max);
+        if (onChange != null) onChange([clampedNewLowerValue, clampedNewUpperValue]);
+      }
+    },
+    [value, min, max, onChange, range, onWheelDisabled],
+  );
+
+  useEffect(() => {
+    const sliderElement = sliderRef.current;
+    if (sliderElement) {
+      sliderElement.addEventListener("wheel", handleWheelEvent, { passive: false });
+    }
+  }, [handleWheelEvent]);
```

Comment:
I was a little surprised about this solution.
In my view the dependency that makes sense would be `sliderRef.current`. I read that components are not rerendered if refs change, which makes sense because the concepts behind refs is to reference objects that are not needed for rendering (according to [react.dev](https://react.dev/reference/react/useRef). I thought that hooks would still be updated, but this must be the wrong assumption.